tools/stringify_file.py

```diff
@@ -22,9 +22,9 @@
 
 # msvc string literal maximum length 16380
 # https://docs.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/compiler-error-c2026?view=msvc-170
-MAX_STRING_LITERAL = 16000
+MAX_STRING_LITERAL = 1600000000000
```

Comment:
`casts.cu` is becoming very long, and this automatically break in the middle of a macro definition. Resulting code like:

```C++
std::string code = R"(
#define X() \
    bla\
"
R"(
bla\
bla
"
```

which lead to generated code

```CUDA
#define X() \
  bla\

  bla\
  bla
```
which does not work.